pkg/volume/volume.go

```diff
@@ -63,6 +63,10 @@ type Attributes struct {
 	ReadOnly        bool
 	Managed         bool
 	SupportsSELinux bool
+	// This will be set if a volume has a GID annotation which is
+	// useful if the volume does not support ownership management
+	// and can only be read as that GID
+	VolumeGID *int64
```

Comment:
per our IRC chat, i think there is agreement that using volumeOptions is better than Attributes.  The kubelet has access to the pod and options here:  https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/volumes.go#L147

we then wouldn't need to modify all the plugins and the behavior is limited to the persistent_claim plugin.